src/mutations/self/index.ts

```diff
@@ -18,3 +18,4 @@ export * from "./useUpdateSelfImage";
 export * from "./useUpdateSelfLead";
 export * from "./useUpdateSelfNotificationPreferences";
 export * from "./useUpdateSelfPushDevice";
```

Comment:
Should be updated to export * from "useAddSelfInterests"

Reply:
done